packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -77,24 +77,21 @@ export class AtomicInsightSearchBox {
   public render() {
     return (
       <SearchBoxWrapper disabled={this.disableSearch}>
+        <atomic-icon part="submit-icon" icon={SearchIcon} class="w-4 h-4" />
```

Comment:
For my above comment

```suggestion
        <atomic-icon part="submit-icon" icon={SearchIcon} class="w-4 h-full my-auto mr-0 ml-4" />
```

(I think, please verify :D )

Reply:
Thanks for the suggestion, will try that.